rclcpp/test/rclcpp/executors/test_static_single_threaded_executor.cpp

```diff
@@ -50,7 +50,6 @@ TEST_F(TestStaticSingleThreadedExecutor, check_unimplemented) {
   auto node = std::make_shared<rclcpp::Node>("node", "ns");
   executor.add_node(node);
 
-  EXPECT_THROW(executor.spin_some(), rclcpp::exceptions::UnimplementedError);
```

Comment:
It's good that you fixed this test, but can you please add some tests for the working version of `spin_some`?  We also want to make sure we cover all of the lines, so be sure to add tests for when things aren't initialized, and when `spin_some` is called inside of `spin_some`.

Reply:
Thanks for the review. I'll add the tests soon

Reply:
@mauropasse friendly ping

Reply:
@ivanpauno thanks for the reminder. As soon as I have time I'll continue this work, currently I'm working on the **EventsExecutor**.

Reply:
I added the `StaticSingleThreadedExecutor` to the list of executors tested for the different spin variations